core/templates/base-components/warnings-and-alerts.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="oppia-toast-container toast-top-center">
+  <div *ngFor="let warning of (this.alertsService.warnings | limitTo:5)"
```

Comment:
`this.alertsService`? Shouldn't it just be alertsService?

Reply:
done

Reply:
done